specification/monitor/resource-manager/microsoft.insights/2016-09-01/serviceDiagnosticsSettings_API.json

```diff
@@ -181,7 +181,7 @@
               "$ref": "#/parameters/ApiVersionParameter"
             },
             {
-                "name": "serviceDiagnosticSettingsResource",
+                "name": "parameters",
```

Comment:
That is because I changed the wrong file... I need to change the example file, not the spec file.
I am on this.